docs/EVG.md

```diff
@@ -460,7 +460,7 @@ If the combination of names or the organization name by itself exceeds 64 charac
 
 **Required/Optional:**  Deprecated (Discouraged, but not prohibited)
 
-**Contents:**   If present, this field MUST contain a single Domain Name(s) owned or controlled by the Subject and to be associated with the Subject's server.  Such server MAY be owned and operated by the Subject or another entity (e.g., a hosting service).  Wildcard certificates are not allowed for EV Certificates except as permitted under Appendix F.
+**Contents:**   If present, this field MUST contain a single Domain Name(s) owned or controlled by the Subject and to be associated with the Subject's server.  Such server MAY be owned and operated by the Subject or another entity (e.g., a hosting service).  Wildcard Domain Names are not allowed for EV Certificates, except as permitted under Section 11.7.1.
```

Comment:
NB: As covered in the EVGs definitions, it includes by reference the BRs' definitions. Ballot 202 aligned the terminology, but the EVGs weren't updated to take advantage of the Proper Definitions